### PR TITLE
[ng] src reorg of acx codelab

### PR DESCRIPTION
**Pull request by chalin**
The corresponding code changes are covered by https://github.com/dart-lang/one-hour-codelab/pull/42

Also cleaned out some links to `...corp.google.com/filiph/acx...`

Contributes to #713

### CODE REVIEW

File: src/_codelabs/angular_components/1-base.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 layout: codelab
 title: "Step 1: Get to Know the Software"
 description: "Get the base app for the codelab, and play with the AngularDart Component demo."
-snippet_img: /codelabs/angular_components/images/cartoon.jpeg
```
Comment (kwalrath) [inline src/_codelabs/angular_components/1-base.md]: Why remove the snippet_img definitions?

Comment (chalin) [inline src/_codelabs/angular_components/1-base.md]: Unless I'm mistaken, it is because they don't contribute anything to those pages.

Comment (chalin) [inline src/_codelabs/angular_components/1-base.md]: (I.e., do you see the `snippet_img` page variable used anywhere?)

Comment (kwalrath) [inline src/_codelabs/angular_components/1-base.md]: That might be a bug in the site implementation. The snippet image is supposed to be used for social sharing.

Comment (kwalrath) [inline src/_codelabs/angular_components/1-base.md]: It should show up in a meta tag. Something like this:

`<meta property="og:image" content="/codelabs/angular_components/images/cartoon.jpeg">`

Comment (chalin) [inline src/_codelabs/angular_components/1-base.md]: Currently, the "og:image" property is essentially set to: `https://webdev.dartlang.org/assets/dart-webdev-logo.png`
See `view-source:https://webdev.dartlang.org/codelabs/angular_components`.

Comment (kwalrath) [inline src/_codelabs/angular_components/1-base.md]: I've filed a bug (https://github.com/dart-lang/site-www/issues/305). Let's keep the `snippet_img` fields in there.

**Comment by chalin** [comment]
> LGTM once you restore the snippet_img fields

Done. This is ready for your re-review.

**Pull request merged**